input/pagecontent/reference.md

```diff
@@ -987,16 +989,16 @@ For a path to a code within a code system, use this syntax:
 
 #### Defining Extensions
 
-To define an extension, the declaration `Extension` is REQUIRED, the keywords `Id`, `Title`, and `Description` are RECOMMENDED, and `Parent` is OPTIONAL. Extensions can also inherit from other extensions, but if the `Parent` keyword is omitted, the parent is assumed to be FHIR's [Extension element](https://hl7.org/fhir/R5/extensibility.html#extension). Extension metadata that does not have a dedicated keyword may be specified using assignment rules with [caret paths](#caret-paths) (e.g., `^url`, `^status`, `^purpose`).
+To define an extension, the declaration `Extension` is REQUIRED, the keywords `Id`, `Title`, and `Description` are RECOMMENDED, and `Parent` is OPTIONAL. Extensions MAY also inherit from other extensions, but if the `Parent` keyword is omitted, the parent SHALL be assumed to be FHIR's [Extension element](https://hl7.org/fhir/R5/extensibility.html#extension). Extension metadata that does not have a dedicated keyword MAY be specified using assignment rules with [caret paths](#caret-paths) (e.g., `^url`, `^status`, `^purpose`).
 
-An extension can have either a value (i.e. a value[x] element) or sub-extensions, but not both. To create a simple extension, the value[x] element should be constrained. To create a complex extension, the extension array of the extension MUST be sliced (see [Contains Rules for Extensions](#contains-rules-for-extensions)).
+An extension MAY have either a value (i.e. a value[x] element) or sub-extensions, but MUST NOT have both. To create a simple extension, the value[x] element SHOULD be constrained. To create a complex extension, the extension array of the extension MUST be sliced (see [Contains Rules for Extensions](#contains-rules-for-extensions)).
```

Comment:
I think it's possible to create a simple extension by simply prohibiting `extension`. In this case we have not constrained value, but it's the only option available -- and there is an invariant saying that either value or extension must exist -- so that effectively requires the value.

Reply:
Good point!